### ISSUE TITLE
Status of this project?

### ISSUE THREAD

**Issue by enumag** [Rich-Harris/buble#27]
I found this Bublé as a light-weight alternative to babel and I like the concept. However I'm very confused about the GitLab and GitHub repos. From the [FAQ](https://buble.surge.sh/guide/#faqs) It looks like you prefer GitLab?

From the status of the GitLab repo I thought the project was dead with the last commit being 9 months old. I only found this repository here by chance. Since there is a newer release here I take it the project is not dead after all and you sometime recently changed your mind about GitLab vs GitHub?

If this repo is the intended future of this project could you add a note to readme of the old repo and update the guide to link here and remove confusing question from FAQ? Or is there some repository for the guide where I could send a PR?

And maybe you should consider creating a buble organization on GH. It's generally a better practice for open-source projects than to have the repo under your personal account.

**Comment by Rich-Harris** [comment]
The project is still alive, though I'll admit it's not as actively developed as would be ideal. This is the correct repo (I tried using GitLab, but frankly GitHub is better in every way that matters — UI, performance, community etc), and issues and PRs will eventually be moved over here instead.

> And maybe you should consider creating a buble organization on GH. It's generally a better practice for open-source projects than to have the repo under your personal account.

I do that for some of my other projects, but it's basically just this repo and one for https://buble.surge.sh, so I don't think there's any value in an org.

I've added a note to the README on GitLab making it clear that it's an abandoned repo! Thanks
